docs/src/man/adding_data.md

```diff
@@ -101,8 +101,11 @@ res[1:2]
 ```
```

Comment:
Related to this, I wonder if if could make things a little bit easier to use a [named tuple](https://docs.julialang.org/en/v1/manual/types/#Named-Tuple-Types) for the output of the plot ?
That way, accessing the data stored in the output might be a little bit easier, and the user would not need to refer to the documentation each time to find the order ? (But maybe that is on purpose ?)

Reply:
That's an excellent idea!

Reply:
I'll do it when I get time. But feel free to go ahead with this change if you have the energy time and interest!

Reply:
Note that this change is possibly breaking, as apparently one cannot get several values of a named tuple in one go:
```julia
julia> res = (0.3, 9, "34")
(0.3, 9, "34")

julia> res[1]
0.3

julia> res[1:2]
(0.3, 9)

julia> res = (xmin = 0.3, xmax = 9, toto = "34")
(xmin = 0.3, xmax = 9, toto = "34")

julia> res[1]
0.3

julia> res[1:2]
ERROR: MethodError: no method matching getindex(::NamedTuple{(:xmin, :xmax, :toto), Tuple{Float64, Int64, String}}, ::UnitRange{Int64})
Closest candidates are:
  getindex(::NamedTuple, ::Int64) at /Applications/Julia-1.7.app/Contents/Resources/julia/share/julia/base/namedtuple.jl:126
  getindex(::NamedTuple, ::Symbol) at /Applications/Julia-1.7.app/Contents/Resources/julia/share/julia/base/namedtuple.jl:127
  getindex(::NamedTuple, ::Tuple{Vararg{Symbol}}) at /Applications/Julia-1.7.app/Contents/Resources/julia/share/julia/base/namedtuple.jl:128
  ...
Stacktrace:
 [1] top-level scope
   @ REPL[20]:1
```

It's easy enough to fix here in the doc, but there might be other uses elsewhere ?
And the change would not be backward compatible. Or maybe I am missing something.

Reply:
bummer. If it's a breaking change, then it would have to version 2.x according to [semantic versioning](https://semver.org). It's a change worth doing though, at some point.